### ISSUE TITLE
mkudfs claims device is mounted or mkudffs is already running

### ISSUE THREAD

**Issue by callegar** [pali/udftools#41]
Hi,

I am getting this, trying to create a filesystem on a CDRW disk. Any clue?

```
mkudffs --media-type=cdrw /dev/pktcdvd/pktcdvd1 
mkudffs: Error: Cannot open device '/dev/pktcdvd/pktcdvd1': Device is mounted or mkudffs is already running
```

**Comment by pali** [comment]
So is device mounted or locked by any other application?

**Comment by callegar** [comment]
Apparently not. I had just completed the `cdrwtool -d /dev/sr1 -q`.
Any way to check?

**Comment by pali** [comment]
Some basic commands for root which may help:

    ls -l -a /dev/pktcdvd/pktcdvd1
    ls -l -a /dev/sr1
    grep /dev/pktcdvd/pktcdvd1 /proc/mounts
    grep /dev/sr1 /proc/mounts
    lsof /dev/pktcdvd/pktcdvd1
    lsof /dev/sr1
    fuser /dev/pktcdvd/pktcdvd1
    fuser /dev/sr1

**Comment by callegar** [comment]
As a matter of fact, I have this:
```
sudo lsof | grep pktcdvd
pktcdvd0  7024                            root  cwd       DIR              253,1       4096          2 /
pktcdvd0  7024                            root  rtd       DIR              253,1       4096          2 /
pktcdvd0  7024                            root  txt   unknown                                          /proc/7024/exe
```
but
```
ps -auxww | grep 7024
root        7024  0.0  0.0      0     0 ?        S<   16:27   0:00 [pktcdvd0]
```
Isn't this the very kernel module providing pktcdvd0?

**Comment by pali** [comment]
Can you provide also output from those grep and ls commands?

**Comment by pali** [comment]
(closing for inactivity)

**Comment by callegar** [comment]
Please see also bug #47. Newest udftools does not give this issue, but filesystems created by mkudffs on cdrw is unusable. Probably this is because cdrwtools makes itself the udf filesystem.

**Comment by pali** [comment]
Probably this issue was fixed by commit c0f4d4c29a7715eeb457722037b7ec4bd37bdb63
